js/5etools-main.js

```diff
@@ -107,6 +108,9 @@ const betteR205etoolsMain = function () {
 			"source",
 			"entries",
 		],
+		"deity": [
+			"name",
```

Comment:
should be `"name", "source", "pantheon"` ("pantheon" being an important third part of the UID for deities) to match the others